readme.md

```diff
@@ -48,6 +49,7 @@ For browser usage, we recommend [Ky](https://github.com/sindresorhus/ky) by the
 - [Plugins](documentation/lets-make-a-plugin.md)
 - [Used by 4K+ packages and 1.8M+ repos](https://github.com/sindresorhus/got/network/dependents)
 - [Actively maintained](https://github.com/sindresorhus/got/graphs/contributors)
+- [Trusted by many companies](#widely-used)
```

Comment:
Feel free to replace this with any text. I haven't thought of anything better yet :)

Reply:
Looks good to me.